docker/README.md

```diff
@@ -0,0 +1,104 @@
+
+
+## How to use?
+
+### First time (assuming you already cloned your fork)
+
+ 1. Go to the docker directory `cd /path/to/miaou/docker` and type `docker-machine-dev` (if on Windows) or `./docker-machine-dev.sh` (else).
+If you followed the [Requirements](#requirements), after a few seconds you will be landed in the VM where you will be able to run `docker` commands.
+
+ 2. Now that you are in your VM, you can build the image `docker-compose` with the alias: 
+	```
+	install-docker-compose
+	```
+ 
+ 3. Then, again, `cd` to the docker directory (the VM shared your `C:\Users\` (windows) or `/Users/` (os x) folder as `/c/Users/` or `/Users/` --- see [complete reference here](https://github.com/boot2docker/boot2docker#virtualbox-guest-additions)).
+ 
+ 4. Build and run the whole Miaou environment with:
+	```
+	docker-compose run --service-ports miaou
+	```
+It will leave you in the Miaou container at `/var/www/html` where you will be able to manage everything like if you were on a server with everything installed.
+If you `exit` this container, it will be stopped then you'll have 2 solutions to come back in it:
+    - Remove the container with `docker rm docker_miaou_run_1` then run it again with `docker-compose run --service-ports miaou` 
+    - Restart the container with `docker start docker_miaou_run_1` then execute `bash` in it with `docker exec -it docker_miaou_run_1 bash`
+ 
+ 5. Once you have all the containers running, you'll have to create all the tables in your database. The whole database is running in its own container.
+In order to be able to run SQL commands, type `docker exec -it docker_postgres_1 psql -U miaou -w`.
+Now that you have a prompt for `miaou` psql user, copy/paste the content of the SQL creation file located at `sql/postgres.creation.sql` in Miaou sources, then exit the container with `\q`.
+
+ 6. Copy the `docker/config.js` preconfigured file to the root of the Miaou sources.
+
+ 7. Now to be able to log in Miaou, you have to set a OAuth provider (here we'll use Google which is really easy to configure).
+First, go to `https://console.developers.google.com/project`, create a new project, then go to `APIs & auth` > `Credentials` to `Create new OAuth Client ID`.
+Select `Web application` as *Application type*, set `http://miaou.dev` as *Authorized Javascript origins* and `http://miaou.dev/auth/google/callback` as *Authorized redirect URIs*.
+Now you'll be given a *Client ID* and a *Client secret* that you'll have to copy/paste in the `config.js` file.
+ 
+ 8. At this point, you should be able to start Miaou. 
+Go back to the Miaou container (see step 4. if you exitted the container) and run the application with `nodemon main.js` (thanks to nodemon, the server will restart automatically after each file modification).
+ 
+ 9. Finally, update your own **hosts file** (e.g. `C:\Windows\System32\drivers\etc\hosts` on Windows) to bind the VM IP address to `miaou.dev`.
```

Comment:
I thought only Windows users wouldn't know where this file is located. Don't you think so?